src/type_.rs

```diff
@@ -1536,7 +1533,7 @@ pub fn register_import(
 
             // Determine local alias of imported module
             let module_name = match &as_name {
-                None => module[module.len() - 1].clone(),
+                None => module.last().unwrap().clone(),
```

Comment:
Let's use `.gleam_expect` here.

Actually it would be good to disallow use of unwrap if possible.

Reply:
Done